### PR TITLE
EOL for Rails < 5.2 and Ruby < 2.3.0

### PR DESCRIPTION
**Pull request by pustovalov**
### Ruby
https://endoflife.software/programming-languages/server-side-scripting/ruby

Ruby 2.2 
Release date
December 25, 2014

End of life
March 31, 2018

Ruby 2.3
December 25, 2015 - March 31, 2019

Ruby 2.4
December 25, 2016 - March 31, 2020

### Rails
Rails 6 
Release date
Aug 16, 2019
https://github.com/rails/rails/releases/tag/v6.0.0

https://guides.rubyonrails.org/maintenance_policy.html

```
For severe security issues all releases in the current major series, and also the last release in the previous major series will receive patches and new versions. The classification of the security issue is judged by the core team.
Currently included series: 6.0.Z, 5.2.Z.
```

### CODE REVIEW

**Comment by smondal** [comment]
I think we should wait until end of life date reached. Lots of application still using Rails 4.2.

**Comment by rafaelfranca** [comment]
> I think we should wait until end of life date reached. Lots of application still using Rails 4.2.

It was reached a few months back when Rails 6.0.0 was released.

**Comment by connorshea** [comment]
Rails 5.0 and 5.1 were also EOL'd, for what it's worth.

**Comment by pustovalov** [comment]
before:
```
Ran for 20 min
Total time 48 min 5 sec
```

after:
```
Ran for 10 min 41 sec
Total time 23 min 3 sec
```

File: webpacker.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
     "changelog_uri"   => "https://github.com/rails/webpacker/blob/v#{Webpacker::VERSION}/CHANGELOG.md"
   }
 
-  s.required_ruby_version = ">= 2.2.0"
+  s.required_ruby_version = ">= 2.4.0"
 
   s.add_dependency "activesupport", ">= 4.2"
```
Comment (rafaelfranca) [inline webpacker.gemspec]: This also need to be updated

**Comment by pustovalov** [comment]
@gauravtiwari can you also check this PR?

**Comment by gauravtiwari** [comment]
thanks @pustovalov

**Pull request merged**